packages/select/README.md

```diff
@@ -49,11 +50,42 @@ class MyApp extends React.Component {
 
 > NOTE: In order to get access to the value, you must add an `onChange` handler, which accepts an event and updates the value of the select as shown above.
 
+#### Enhanced Select
+
+React Select provides a Material styled select, instead of the default native drop down.
+
+```js
+import React from 'react';
+import Select, {Option} from '@material/react-select';
+
+class MyApp extends React.Component {
+  state = {value: 'pomsky'};
+
+  onEnhancedChange = (index, item) => (
+    this.setState({value: item.getAttribute('data-value')});
+  );
+
+  render() {
+    return (
+      <Select
+        enhanced
+        label='Choose Dog'
+        value={this.state.value}
+        onEnhancedChange={this.onEnhancedChange}
+      >
+        <Option value='pomsky'>Pomsky</Option>
+        <Option value='goldenDoodle'>Golden Doodle</Option>
+      </Select>
+    );
+  }
+}
+```
+
 #### Shorthand options
 
-If the option elements do not require anything unique, you can omit passing `this.props.children` and set the `option` attribute.
+If the option elements do not require anything unique, you can omit passing `this.props.children` and set the `options` prop.
 
-This shorthand option is most useful when you're select options come in the form of JSON. Typically when building a select, the options will be backed by an array of objects or strings (either from an API endpoint or a JSON File). If the label/value key names, you can run a `map` function over your options.
+This options shorthand is most useful when you're select options come in the form of JSON. Typically when building a select, the options will be backed by an array of objects or strings (either from an API endpoint or a JSON File). If the label/value key names, you can run a `map` function over your options.
```

Comment:
`you're` -> `your`
Typically when -> Typically, when

`If the label/value key names, you can run a map function over your options`...... what?